common/filetoolkitwithundo.h

```diff
@@ -59,6 +58,18 @@ class FileToolkitWithUndo
 protected:
     ReturnCode createSymLink(const std::string &source, const std::string &destination);
 
+    /*
+     * @brief Writes to a file (and optionally create it)
+     */
+    ReturnCode writeToFile(const std::string &path, const std::string &content);
+
+    /*
+     * @brief Creates a file cleanup handler for a specific file.
+     *
+     * This is useful if one creates a file in some other way and want it deleted later.
+     */
+    void watchFile(const std::string &path);
```

Comment:
If this method is purely used for deleting files later then I think we can find a better name :)